README.md

```diff
@@ -27,42 +27,79 @@ const Example = () => (
 
 This repo has to be run with https://github.com/openworklabs/aragon
 
-Clone this repo, install dependencies, link, and build: <br />
-`git clone git@github.com:AutarkLabs/aragon-profile.git`<br />
-`cd aragon-profile`<br />
-`npm i`<br />
-`npm run prepare-local-dev`<br />
-`npm link`<br />
-`npm build`<br />
+Clone this repo, install dependencies, link, and build:
+`git clone git@github.com:AutarkLabs/aragon-profile.git`
+`cd aragon-profile`
+`npm i`
+`npm run prepare-local-dev`
+`npm link`
+`npm run build`
```

Comment:
These are not showing up on separate lines in the rendered html.

Reply:
We can fix that by adding two spaces at the end of each line, or maybe better, putting them all in a code block. Indent each line four spaces, or wrap the block in triple backticks. Making this a bulleted or number list might also look nice.